specutils/manipulation/resample.py

```diff
@@ -0,0 +1,208 @@
+import abc
```

Comment:
Shouldn't this file be completely gone from this PR? (I don't see it used anywhere else now)